src/sentry/static/sentry/app/views/settings/organizationAuditLog/auditLogList.jsx

```diff
@@ -125,18 +85,21 @@ class AuditLogList extends React.Component {
                       <Note>{entry.note}</Note>
                     </NameContainer>
                   </UserInfo>
-                  <div>{entry.event}</div>
                   <div>
+                    <MonoDetail>{entry.event}</MonoDetail>
```

Comment:
I wouldn't right align or monospace this column. The data isn't numeric values, and doesn't benefit much from monospacing.

Reply:
I think the monospace makes sense since it's a key

Reply:
But I agree it should be left aligned.